src/route-utils.js

```diff
@@ -138,6 +138,12 @@ export function parseRoute({params, routes, location}) {
 class FilterState {
   constructor(state) {
     this.state = state;
+    if (this.state.repoInfos.length === 0) {
+      this.state.repoInfos.push({
+        repoName: 'coala',
```

Comment:
as you are not using `issues.json` to determine the default repo list, this needs to be obtained from the build environment variable `REPOSITORIES`, which e.g. has a value `coala:coala|coala-bears|documentation|projects`

To make that work, you'll need to push that value from the build environment into the generated website so that it can be loaded by the client.

Reply:
@jayvdb Done sir